meson.build

```diff
@@ -0,0 +1,177 @@
+project('waltham', 'c',
+        version: '0.0.0',
+        license: ['MIT'],
```

Comment:
`MIT/Expat` I believe, otherwise it is quite ambiguous.
We should also have `default_options: ['c_std=gnu99', 'warning_level=2']`. Unfortunately I haven't found documentation for warning_level, but I'd assume it replaces most if not all of the manual warning flags we used to add.

Reply:
Looks like `warning_level=2` means `-Wall -Wextra` and nothing else.